atlp-rwanda/alpha-ec-be
# Ft user registration 187300143

### issue in referencing?

### Questions:

#### What does this PR do?

User Registration

#### Description of Task to be completed?

- Users once registered via the registration endpoint, should receive a JWT to be required on all subsequent calls to all other endpoints that require authentication.

Users should receive a descriptive error message when they enter invalid user parameters in an attempt to signup.

#### How should this be manually tested?

- run `npm install` 
- run `npm run migrate`
- run `npm run seed`
- run `npm run dev`
- check the [swagger documentation ](http://localhost:3000/swagger/#/users/post_api_users_register)

#### Any background context you want to provide?

-dummy data are provided before having the real data

#### What are the relevant pivotal tracker/Trello stories?

- id of the project
#187300143

#### Questions:
